angular/src/app/auth/auth.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.css']
+})
+export class AuthComponent implements OnInit {
+
+  constructor () { }
+
+  ngOnInit () {
+  }
+
+  onFenixLogin () {
+    sessionStorage.setItem('user', '')
```

Comment:
This won't work on any apple device in incognito mode

Reply:
Thanks, I didn't know. It's true, also localStorage.

Some apps disable access if user as incognito mode enable: https://protonmail.com/support/knowledge-base/enable-sessionstorage/

Suggestion: Maybe alert the user to disable incognito mode to perform login.

Reply:
I ended up solving this my creating a Service as an abstraction layer on top of storage. Something like this
```
import { Injectable } from '@angular/core';

@Injectable()
export class StorageService {
  private internalStorage;

  public setItem(key, value) {
    try {
      sessionStorage.setItem(key, value)
    } catch() {
      internalStorage[key] = value;
    }
  }

  public getItem(key) {...}
  public removeItem(key) {...}
  {etc}
}
```

If it's possible to use the session or the local storage we just use it, if not we just store stuff in memory. Of course if the user refreshes the page it will be logged out.

Reply:
Yes, it's a good solution. 

Thanks for the help @FabioAntunes ! 💯

Reply:
Why not use a cookie?

Reply:
We can use cookies. But we need to change the API to accept them and use it to authenticate.
The actual implementation use sessionStorage.